.github/workflows/jruby.yml

```diff
@@ -0,0 +1,31 @@
+name: jruby
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - staging
+      - trying
+
+jobs:
+  warbler:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Setup ruby
+        uses: eregon/use-ruby-action@v1
+        with:
+          ruby-version: jruby-9.2.9.0
+
+      - name: Install local bundler
+        run: bin/rake install:local
+
+      - name: Run a warbler project
+        run: |
+          cd spec/realworld/fixtures/warbler
+          bundle install
+          bundle exec warble
+          java -jar warbler.jar
```

Comment:
great!

I think you can just add a second inner gem `demo-test` and have this same workflow and leverage this test setup for https://github.com/rubygems/bundler/issues/7611

Reply:
Thanks, I'll keep that in mind :+1: